python_modules/dagster/dagster/_utils/test/schedule_storage.py

```diff
@@ -710,61 +766,51 @@ def test_auto_materialize_asset_evaluations(self, storage):
                     num_skipped=0,
                     num_discarded=0,
                 ),
-                AutoMaterializeAssetEvaluation(
-                    asset_key=AssetKey("asset_two"),
-                    partition_subsets_by_condition=[
-                        (
-                            AutoMaterializeRuleEvaluation(
-                                rule_snapshot=AutoMaterializeRule.materialize_on_missing().to_snapshot(),
-                                evaluation_data=None,
-                            ),
-                            None,
-                        )
-                    ],
-                    num_requested=1,
-                    num_skipped=0,
-                    num_discarded=0,
-                ),
             ],
         )
 
         res = storage.get_auto_materialize_asset_evaluations(
             asset_key=AssetKey("asset_one"), limit=100
         )
+        assert len(res) == 2
+        assert res[0].evaluation_id == 11
+        assert res[1].evaluation_id == 10
+
+        res = storage.get_auto_materialize_asset_evaluations(
+            asset_key=AssetKey("asset_one"), limit=1
+        )
         assert len(res) == 1
-        assert res[0].evaluation.asset_key == AssetKey("asset_one")
-        assert res[0].evaluation_id == 10
-        assert res[0].evaluation.num_requested == 0
+        assert res[0].evaluation_id == 11
 
         res = storage.get_auto_materialize_asset_evaluations(
-            asset_key=AssetKey("asset_two"), limit=100
+            asset_key=AssetKey("asset_one"), limit=1, cursor=11
         )
         assert len(res) == 1
-        assert res[0].evaluation.asset_key == AssetKey("asset_two")
         assert res[0].evaluation_id == 10
-        assert res[0].evaluation.num_requested == 1
 
-        res = storage.get_auto_materialize_evaluations_for_evaluation_id(evaluation_id=10)
+        # add a mix of keys - one that already is using the unique index and one that is not
```

Comment:
wow github really struggled with identifying what i actually changed here, i just added this case at the end and put the first part in a for loop to run twice